src/expr-parser/Cargo.toml

```diff
@@ -10,7 +10,7 @@ publish = false
 mz-expr = { path = "../expr" }
 mz-ore = { path = "../ore" }
 mz-repr = { path = "../repr" }
-proc-macro2 = { version = "1.0.47", features = ["span-locations"] }
+proc-macro2 = { version = "1.0.60", features = ["span-locations"] }
```

Comment:
Why is the `Cargo.lock` mentioning `1.0.66` but we are using `1.0.60` here (and in other locations)? Is this correct?

Reply:
Yeah, it has to do with how Rust/Cargo [resolves dependency versions](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html). tl;dr specifying `1.0.60` means we can depend on any version `>= 1.0.60` and `< 2.0.0`. So here Cargo picks the newest version of the dependency available which is `1.0.66`